source/howto/Volume.rst

```diff
@@ -0,0 +1,426 @@
+Volumes in OpenStack 

```

Comment:
This looks like it should be a title (a = line before and after)

Reply:
Added the title line